subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -583,6 +583,53 @@ int nct_init(void)
 	return mqtt_init();
 }
 
+#if defined(CONFIG_NRF_CLOUD_STATIC_IPV4)
+/* Partly copy/paste from zephyr/subsys/net/ip/utils.c
+ * Cannot be used when BSD library selects NET_RAW_MODE */
+int af_inet_addr(const char *src, void *dst)
```

Comment:
Too bad we can't use Zephyr function here :/ Disabling NET_RAW_MODE just to get access to this function does not sound like a good idea though...

Nontheles, I'd prefer to stick to the standard `inet_pton` function definition, just as Zephyr implementation does, i. e.:
```
int inet_pton(int af, const char *src, void *dst);
```
In case NET_RAW_MODE is improved in the future, or we will no longer want to use it, switching to different implementation should be straightforward. If you do not want to implement IPv6, just return an error code in case of `AF_INET6`.
And I'd include `pton` (presentation to network) somewhere in the function name, just to make it clear what kind of function we deal with.

Reply:
I have noticed that there other useful functions in `net/ip/utils.c` that are disabled in `NET_RAW_MODE`; that's unfortunate :/